materialize-stepper.js

```diff
@@ -5,274 +5,309 @@
+    var order = step_num > prev_active ? 1 : 0;
+    if (active.hasClass('feedbacking')) $this.destroyFeedback();
+    active.closeAction(order);
+    step.openAction(order, function () {
+        if (settings.autoFocusInput) step.find('input:enabled:visible:first').focus();
+        $this.trigger('stepchange').trigger('step' + (step_num + 1));
+        if (step.data('event')) $this.trigger(step.data('event'));
+        if (callback) callback();
+
+        if ($this.closest('ul').hasClass('horizontal')) {
+            newHeight = 0;
+            padding = 140;
+
+            $('.stepper.horizontal').find('.step.active').find('.step-content > div').each(function (index, value) {
+                console.log(value)
+                newHeight = newHeight + parseInt($(value).css('height'));
+            });
+
+            newHeight += padding;
+
+            $('.stepper.horizontal').animate({
+                height: newHeight
+            },
+                300);
+            //  openable.slideDown(300, "easeOutQuad", callback);
+
+        }
+
+    });
 };
 
-$.fn.closeAction = function(order, callback) {
-   var closable = this.removeClass('active').find('.step-content');
-   if(window.innerWidth < 993 || !this.closest('ul').hasClass('horizontal')) {
-      closable.stop().slideUp(300,"easeOutQuad", callback);
-   } else {
-      if(order==1) {
-         closable.animate({left: '-100%'},function(){closable.css({display: 'none', left: '0%'}, callback);});
-      } else {
-         closable.animate({left: '100%'},function(){closable.css({display: 'none', left: '0%'}, callback);});
-      }
-   }
+$.fn.closeAction = function (order, callback) {
+    var closable = this.removeClass('active').find('.step-content');
+    if (window.innerWidth < 993 || !this.closest('ul').hasClass('horizontal')) {
+        closable.stop().slideUp(300, "easeOutQuad", callback);
+    } else {
+        if (order == 1) {
+            closable.animate({ left: '-100%' }, function () { closable.css({ display: 'none', left: '0%' }, callback); });
+        } else {
+            closable.animate({ left: '100%' }, function () { closable.css({ display: 'none', left: '0%' }, callback); });
+        }
+    }
 };
 
-$.fn.openAction = function(order, callback) {
-   var openable = this.removeClass('done').addClass('active').find('.step-content');
-   if(window.innerWidth < 993 || !this.closest('ul').hasClass('horizontal')) {
-      openable.slideDown(300,"easeOutQuad", callback);
-   } else {
-      if(order==1) {
-         openable.css({left: '100%', display: 'block'}).animate({left: '0%'}, callback);
-      } else {
-         openable.css({left: '-100%', display: 'block'}).animate({left: '0%'}, callback);
-      }
-   }
+$.fn.openAction = function (order, callback) {
+    var openable = this.removeClass('done').addClass('active').find('.step-content');
+    if (window.innerWidth < 993 || !this.closest('ul').hasClass('horizontal')) {
```

Comment:
I think the check on the ul having the horizontal class should be testing for true, not false here?

`if (window.innerWidth < 993 || this.closest('ul').hasClass('horizontal')) {`